common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -118,9 +118,19 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-sign-in-gate-centesimus-2",
-    "Show sign in gate to 100% of users on 3rd article view",
-    owners = Seq(Owner.withGithub("vlbee")),
+    "ab-sign-in-gate-main-control",
+    "Show sign in gate to 100% of users on 3rd article view, control audience",
```

Comment:
Should this description be changed? 9% who will never see the gate?

Reply:
you are correct 🙈